test/IRGen/resilience_weakly_linked.swift

```diff
@@ -0,0 +1,20 @@
+// RUN: %empty-directory(%t)
```

Comment:
We already have test/IRGen/weak_import_native.swift, can you add your test case there instead?